### PR TITLE
add volumes configration to attach tasks

### PR DESCRIPTION
**Pull request by vampire-yuta**
Hi!  
I added volumes attached feature.
Please check it.

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -121,6 +121,12 @@ resource "aws_ecs_task_definition" "this" {
   tags                     = var.tags
   task_role_arn            = try(aws_iam_role.ecs_task[0].arn, var.ecs_task_role)
 
+  dynamic "volume" {
+    for_each = var.volumes
```
Comment (sblack4) [inline main.tf:125]: `for_each` only accepts maps or sets of strings https://www.terraform.io/language/meta-arguments/for_each#basic-syntax. `volumes` is list :(

File: variables.tf [hunk 2]
```diff
@@ -101,6 +101,14 @@ variable "network_mode" {
   type        = string
 }
 
+variable "volumes" {
+  default     = null
+  description = "The Volume to attach ecs container. This is a list of maps"
+  type = list(object({
```
Comment (sblack4) [inline variables.tf]:
```suggestion
  type = set(object({
```

**Review by sblack4 — CHANGES_REQUESTED** [review]
Thank you! I think declaring a variable as a set will work but I haven't tested it.

**Comment by vampire-yuta** [comment]
Hi.  😸 
Your suggestion is fixed. How about it?? 👀

**Review by sblack4 — APPROVED** [review]

**Pull request merged**